bodhi/client/__init__.py

```diff
@@ -79,6 +79,8 @@ def new(user, password, **kwargs):
             print_resp(resp, client)
         except bindings.BodhiClientException as e:
             click.echo(str(e))
+        except Exception as AuthError:
```

Comment:
@ankit01ojha if you want you can look at my branch, I think this would handle all command. Maybe you can test it.
https://github.com/cverna/bodhi/blob/handle_wrong_password/bodhi/client/bindings.py#L309